src/web/theme/modules/Tile/Tile.scss

```diff
@@ -1,7 +1,9 @@
-$tilePadding: 16rem;
+$tileWidthPadding: 16rem;
```

Comment:
I don't think all of these variables are necessary, you could have a base `$largeSpacing: 8rem;` and `$baseSpacing: 2rem;` and use multiplications of these.

In any case if you keep your various variables, I would think that Spacing is better than Padding since you could use these dimensions for a variety of rules, not just padding.

Also I would think that vertical / horizontal would be a better fit than width / height